### ISSUE TITLE
NavView: EventTriggerBehavior does not work on ItemInvoked 

### ISSUE THREAD

**Issue by ghanimekki** [Kinnara/ModernWpf#118]
Hello, 
I'm trying to bind ItemInvoked to a command on my ViewModel using Caliburn Micro. However, the event is not triggering. The same thing is true for BackButton.

According to this, https://github.com/Microsoft/XamlBehaviors/issues/142 , it appears to be an issue with TypedEventHandler and .NET complier. The solution would be to "tell the compiler" to expose the events.

I have noticed the same behaviour on other controls (i.e. flyout) from the library. A work arround was to call the command from.. code-behind, but it's not sustainable as the MVVM project gets bigger. Till now, I did face many difficulties getting the library's controls' events to work with Caliburn/hand-written Commands on VM. 

Maybe it would be better to expose the event in the library such as it can be used with EventTriggerBehavior ?
(Unfortunately, I'm not experienced enough with the framework, to come up with better solutions)

Or, maybe expose ItemInvokedCommand and BackCommand on the control ?

P.S : The ItemInvoked event was attaching correctly on mah:HamburgerMenu.

**Comment by Kinnara** [comment]
Where can I get EventTriggerBehavior? I can't find it in either [Caliburn.Micro](https://github.com/Caliburn-Micro/Caliburn.Micro) or [XamlBehaviorsWpf](https://github.com/Microsoft/XamlBehaviorsWpf).

**Comment by ghanimekki** [comment]
I meant EventTrigger, my bad.

I tried adding the rd.xml (runtime directive) file, but I think it's exclusisve to UWP, I have also tried different methods to "convert" the event to a command, it works, but the behavior is not truly that of a command.

I'm now just binding SelectedItem to property, and handling it's change to fire a method on my ViewModel. But for BackRequested event, and other events on, Flyout for example, I have to use code-behind.

Adding cal:Message.Attach="[Event Click] = [Action Remove($dataContext)]" to a control in caliburn, translates to an interactivity EventTrigger to bind to the ViewModel.

 xmlns:i="clr-namespace:System.Windows.Interactivity;assembly=System.Windows.Interactivity"

            <i:Interaction.Triggers>
                <i:EventTrigger EventName="Click">
                    <cal:ActionMessage MethodName="SayHello" />
                </i:EventTrigger>
            </i:Interaction.Triggers>

**Comment by dorincanepa** [comment]
I think this issue was already fixed by @Kinnara  when derive event data classes form EventArgs. #178 
For this test I use mvvmlight 

```
        xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
        xmlns:cmd="clr-namespace:GalaSoft.MvvmLight.Command;assembly=GalaSoft.MvvmLight.Platform"
```

```
 <ui:NavigationView
            AlwaysShowHeader="True"
            IsBackButtonVisible="Collapsed"
            IsTitleBarAutoPaddingEnabled="False"
            IsPaneOpen="True"
            OpenPaneLength="240"
            PaneTitle="Test"
            IsSettingsVisible="False"
            PaneDisplayMode="Left"
            MenuItemsSource="{Binding NavigationViewPluginsItems}"
            FooterMenuItemsSource="{Binding NavigationFooterItems}">

            <i:Interaction.Triggers>
                <i:EventTrigger EventName="ItemInvoked">
                    <cmd:EventToCommand Command="{Binding UpdateCommand}" PassEventArgsToCommand="true" />
                </i:EventTrigger>
	    </i:Interaction.Triggers>

...
```

**Comment by Kinnara** [comment]
Resolved by a23d5bc.
